languages/c/reference/exercise-concepts/two-fer.md

```diff
@@ -1,22 +1,16 @@
 # Concepts of two-fer
 
-```c
-#include "two_fer.h"
-#include <stdio.h>
-
-void two_fer(char *buffer, const char *name) {
-    sprintf(buffer, "One for %s, one for me.", name ? name : "you");
-}
-```
+[Example Implementation](https://github.com/exercism/c/blob/master/exercises/two-fer/src/example.c)
 
 - [Header Files](https://www.gnu.org/software/libc/manual/html_mono/libc.html#Header-Files): used to access the standard library
-- [C Standard Library header files](https://en.cppreference.com/w/c/header): stdio.h is a C standard library header file
 - [Function Definitions](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Function-Definitions): used as the main entry point for the exercise
-- [Function Parameters](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Function-Parameters): buffer and name are passed as arguments
+- [Function Parameters](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Function-Parameters): buffer and name are function parameters
```

Comment:
This is better, but hmmm.... `buffer` and `name` are parameters at one line, but used as arguments at another. Should we be even more specific? 
Perhaps "the definition of the function under test uses parameters"